### PR TITLE
[BUGFIX] added cleanup on navigate to stop contact-form component

### PR DESCRIPTION
**Pull request by michaelzangerle**
__Tasks:__
- [x] done
- [ ] got feedback

__Informations:__

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | none
| Tests pass?   | none
| Fixed tickets | XXX
| Doc           | none

### CODE REVIEW

File: src/Sulu/Bundle/ContactBundle/Resources/public/dist/components/contact-form/main.js [hunk 1]
```diff
@@ -1 +1 @@
...
...x.dom.find("label.hidden","#contact-edit-form");for(a=-1,b=c.length;++a<b;)U.call(this,this.sandbox.dom.parent(c[a]))},U=function(a){var b=this.sandbox.dom.trim(this.sandbox.dom.text(this.sandbox.dom.find("label.hidden",a)));b=b.replace(/\s{2,}/g," "),this.sandbox.dom.attr(this.sandbox.dom.find("label.visible",a),"title",b),this.sandbox.dom.html(this.sandbox.dom.find("label.visible",a),this.sandbox.util.cropMiddle(b,20))};return{initialize:function(){this.initializedFlag=!1,this.$editOverlayContent=null,this.form=null,this.$addOverlay=null,this.dropdownDataArray=[],this.editFieldsData=[],this.data=null,this.options=this.sandbox.util.extend(!0,{},c,this.options),H.call(this),this.render(),n.call(this),o.call(this),this.sandbox.emit(g.call(this)),this.initializedFlag=!0},render:function(){var a=this.sandbox.dom.createElement('<div id="contact-form-options-container" />'),b=this.$find(d.dropdownContainerId);this.sandbox.dom.append(b,a),this.sandbox.start([{name:"dropdown@husky",options:{trigger:b,triggerOutside:!0,el:a,alignment:"right",shadow:!0,toggleClassOn:b,data:[{id:1,name:"public.edit-fields",callback:L.bind(this)},{id:2,name:"public.add-fields",callback:S.bind(this)}]}}])}}});
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContactBundle/Resources/public/dist/components/contact-form/main.js]: why has this changed was it forgotten last time or did you have change it accidental?

**Comment by wachterjohannes** [comment]
@michaelzangerle wrong template! and i wouldnt add the link to a private client project in a public PR!

File: src/Sulu/Bundle/ContactBundle/Resources/public/dist/components/contact-form/main.js (hunk 1 above)
Comment (michaelzangerle) [inline src/Sulu/Bundle/ContactBundle/Resources/public/dist/components/contact-form/main.js]: I did not change it. I think someone forgot to build it and therefore it's in here.

File: src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/form/main.js [hunk 2]
```diff
@@ -387,6 +389,16 @@ define(['config', 'widget-groups'], function(Config, WidgetGroups) {
...
+
+            this.sandbox.on('sulu.router.navigate', this.cleanUp.bind(this));
+        },
+
+        /**
+         * Does some cleanup with aura components
+         */
+        cleanUp: function() {
+            // stop contact form before leaving
+            this.sandbox.stop(constants.editFormSelector);
```
Comment (danrot) [inline src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/form/main.js]: Why is this necessary at all? Something like this should already be executed in the on the navigate event, maybe the structure of the dom elements is wrong, and this is just fixing symptoms?

Comment (michaelzangerle) [inline src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/form/main.js]: The component wont be stopped when only switching tabs and therefore it has to be done manually. DOM structure is fine I checked that already.

Comment (danrot) [inline src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/form/main.js]: Maybe that's also an input for @marcelmoosbrugger, because the components should be stopped on a tab change IMO, so that something like this is not necessary.

Comment (michaelzangerle) [inline src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/form/main.js]: +1
